.github/workflows/main.yml

```diff
@@ -0,0 +1,27 @@
+name: Build & Test
+
+on:
+  push:
+    branches: [main]
```

Comment:
this one too should be `master` i think

Reply:
whoops lol copied from my work scheduling